eZ/Publish/API/Repository/SearchService.php

```diff
@@ -18,6 +18,23 @@
 interface SearchService
 {
     /**
+     * Capability constants for use with {@see ::supports()}.
+     *
+     * Will add constants for highlighting, spellchecking & suggest once there is at least one implementation proving api design.
+     *
+     * Custom feature flags for features not made official yet is possible by prefixing values of the flags with
+     * identifier for engine, and placing the constant on said search engine. However this kind of tight coupling
+     * between api consumer and specific search engines is not officially supported by the api (as it's then not the api)-
+     */
+    const CAPABILITY_SCORING = 1;
```

Comment:
Is there any real value in using integers and not text, besides the saved bytes ? I'd maybe prefer a lowercase version of the constant. Not very important, but may be handy when debugging, and no serious reason not to.

Reply:
> Is there any real value in using integers and not text, besides the saved bytes

In my review I even suggested going deeper with `int`s and use power of 2, so `supports` implementation could look like this:
```php
public function supports($capabilityFlag)
{
    return $this->supportedFeatures & $capabilityFlag;
}
```